### PR TITLE
feat: Add anonymous usage and profiling stats to OpenMetadata using OpenTelemetry

### PR DESCRIPTION
**Pull request by akash-jain-10**
### Describe your changes :

See #1396 

Will close #1397 in favour of this.

#
### Type of change :

- [x] Improvement
- [x] New feature

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: ingestion/setup.py [hunk 1]
```diff
@@ -43,6 +43,9 @@ def get_long_description():
     "requests~=2.26",
     "cryptography",
     "PyYAML",
+    "opentelemetry-exporter-otlp-proto-http>=1.6.2,<1.7",
```
Comment (harshach) [inline ingestion/setup.py:46]: any reason for this to be less than 1.7

File: ingestion/src/metadata/telemetry/openmetadata_telemetry.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        if self.__local_debug_mode or logger.getEffectiveLevel() == logging.DEBUG:
+            provider.add_span_processor(console_span_processor)
+
+        if not self.__local_debug_mode:
+            # create a JaegerExporter
+            jaeger_exporter = JaegerExporter(
+                # configure agent
+                agent_host_name="collect.open-metadata.org",
+                agent_port=6831,
+                # optional: configure also collector
```
Comment (harshach) [inline ingestion/src/metadata/telemetry/openmetadata_telemetry.py:82]: remove all the commented out code

File: ingestion/src/metadata/utils/docker.py [hunk 3]
```diff
@@ -149,4 +177,10 @@ def run_docker(start, stop, pause, resume, clean, file_path):
     except Exception as err:
         logger.error(traceback.format_exc())
```
Comment (harshach) [inline ingestion/src/metadata/utils/docker.py:178]: lets change these to debugs

File: ingestion/setup.py (hunk 1 above)
Comment (akash-jain-10) [reply ingestion/setup.py:46]: This has been ported from previous Pull request. Not sure if we need to update this.
Ref - https://github.com/open-metadata/OpenMetadata/pull/1397/files

**Pull request closed without merging**